src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -696,13 +1030,22 @@ void EIRPlantLoopHeatPump::sizeSrcSideWSHP(EnergyPlusData &state)
             tmpSourceVolFlow = hardSizedSourceSideFlow;
         }
     }
+    if (this->companionHeatPumpCoil) {
+        tmpSourceVolFlow *= this->companionHeatPumpCoil->heatSizingRatio;
+    } else {
+        tmpSourceVolFlow *= this->heatSizingRatio;
+    }
 
     // skipping autosized power section
 
     // register the design volume flows with the plant, only doing half of source because the companion
     // is generally on the same loop
-    PlantUtilities::RegisterPlantCompDesignFlow(state, this->loadSideNodes.inlet, tmpLoadVolFlow);
-    PlantUtilities::RegisterPlantCompDesignFlow(state, this->sourceSideNodes.inlet, tmpSourceVolFlow / 0.5);
```

Comment:
next, if not a heat recovery HP register plant flow rate, otherwise this HP does not add to plant sizing